tmpl/wiki/weekly/2015-09-30.md

```diff
@@ -0,0 +1,189 @@
+30th September 2015: Functoria, Irmin plans, numbering and Pioneers
+
+### Agenda ###
+
+- Functoria
+- Irmin roadmap
+- Version numbering
+- Pioneer Projects
+
+Attendees:
+Reynir Björnsson, Daniel Bünzli, Amir Chaudhry (chair), Justin Cormack,
+Thomas Gazagnaire, David Kaloper, Thomas Leonard, Anil Madhavapeddy,
+Mindy Preston, Dave Scott, Magnus Skjegstad, Guybrush Threepwood and
+Jim Tittlser
+
+
+### Notes ###
+
+#### Functoria ####
+
+*See [mirage/mirage#441][]*
+
+There's been work towards a new EDSL that's captured in a library called
+Functoria.  We discussed this last time and it needed rebasing due to an
+earlier `.xl` patch being merged.
+
+Things have been cleaned up with the patch but it seems there have been some
+issues with Travis, which someone needs to take a closer look at.  Someone
+also needs to review and test the patch (Anil will do this, feedback from
+others also welcome). The examples should also be checked over.
+
+One thing we know is still missing as there's an opportunity to retain the
+shorthand flags we have for `--xen` and `--unix` as part of the configure step.
+ThomasG's not had a chance to look at that yet but would be great to keep them
+if we can.  There's also been some work on related tools we use (cmdliner), to
+add support for environment variables ([dbuenzli/cmdliner@e82fd02][]).
+
+Aside from the code review, there's already a draft blog post prepared (Amir
+will review), as well as a note for the [breaking changes][brk-chg] page of
+the website (a few things may change for some people).
+
+Getting the IPV6 support in was also blocked on the configuration step too.
+Once we've completed the releases around Functoria, we can work on getting an
+IPV6 site up! 
+
+[mirage/mirage#441]: https://github.com/mirage/mirage/pull/441
+[dbuenzli/cmdliner@e82fd02]: https://github.com/dbuenzli/cmdliner/commit/e82fd02842719167c3a04fc7ed0edf9a8036203c
+[brk-chg]: https://mirage.io/wiki/breaking-changes
+
+#### Irmin roadmap ####
+
+There are a number of things we can be doing with Irmin so there's a question
+of how we decide what exactly what we should work on next. It seems people
+have lists of things they'd like to get done but pulling these into a clearer
+plan would be helpful.
+
+ThomasL can compile Irmin to Javascript and and then call it from Javascript.
```

Comment:
"and and"

The Irmin JS bindings are here: https://github.com/talex5/irmin-js
I know nothing about JS, so if someone can show me a better syntax for the examples, I'd love to hear it. Here's my attempt: https://github.com/talex5/irmin-js/blob/master/examples/tests.js